run_tests.sh

```diff
@@ -11,13 +11,17 @@ if [ -f accumulator.pid ]; then
   fi
 fi
 
+if [ -z "$ACC_ENDPOINT" ]; then
+  export ACC_ENDPOINT=http://localhost:3000
+fi
+
```

Comment:
A shorter way to write the same:

```
ACC_ENDPOINT=${ACC_ENDPOINT:-http://localhost:3000}
```

And, no need to export the env var as it is passed to accumulator.js

Reply:
oh yes, I forgot to remove that!